packages/survey-creator-core/src/components/tabs/designer.ts

```diff
@@ -103,30 +106,36 @@ export class TabDesignerViewModel<T extends SurveyModel> extends Base {
   }
 }
 
-export class TabDesignerPlugin<T extends SurveyModel>
-implements ICreatorPlugin
-{
+export class TabDesignerPlugin<T extends SurveyModel> implements ICreatorPlugin {
   public model: TabDesignerViewModel<T>;
+  public propertyGrid: PropertyGridViewModelBase;
   private undoAction: Action;
   private redoAction: Action;
   private surveySettingsAction: Action;
   private saveSurveyAction: Action;
+  private expandAction: Action;
 
   constructor(private creator: CreatorBase<T>) {
     creator.addPluginTab("designer", this);
+    const propertyGridModel = new PropertyGridModel(creator.survey as any as Base, creator);
+    this.propertyGrid = new PropertyGridViewModel(propertyGridModel, creator);
   }
   public activate(): void {
     this.model = new TabDesignerViewModel<T>(this.creator);
     this.undoAction && (this.undoAction.visible = true);
     this.redoAction && (this.redoAction.visible = true);
     this.surveySettingsAction && (this.surveySettingsAction.visible = true);
+    this.creator.showPropertyGrid = true;
```

Comment:
It will show property grid every time a user go to designer.
Let's say, a user hides property grid, go to test tab, comeback and property grid will be show again.
It looks like a bug to me.

Reply:
I fixed the problem.